requirements/common_constraints.txt

```diff
@@ -26,13 +25,6 @@ elasticsearch<7.14.0
 
 # django-simple-history>3.0.0 adds indexing and causes a lot of migrations to be affected
 
-# Cause: https://github.com/openedx/event-tracking/pull/290
-# event-tracking 2.4.1 upgrades to pymongo 4.4.0 which is not supported on edx-platform.
-# We will pin event-tracking to do not break existing installations
-# This can be unpinned once https://github.com/openedx/edx-platform/issues/34586
-# has been resolved and edx-platform is running with pymongo>=4.4.0
```

Comment:
Was this pulled in accidentally?

Reply:
This was introduced in https://github.com/openedx/edx-platform/pull/35734/commits/f9ef3f9649e58a45a53324f9211fd4e888fb9d82. I ran `make compile-requirements` in the cms shell. The command generates those changes automatically. Is this something unexpected? I have re-run the command on this branch and `master` and I get the same result

Reply:
Personally, I always remove any changes like that from the diff before committing, if it's unrelated to the PR in question. Keep things focused on one set of changes. Those dependencies will be updated in a separate PR.

Reply:
Ok, make sense, I will do that :+1:

Reply:
@bradenmacdonald @ormsbee It's already updated here https://github.com/openedx/edx-platform/commit/040ff12430d2418f8bea0c7ec8d2eeb0f734a8e7